src/Drupal/Commands/core/LocaleCommands.php

```diff
@@ -128,4 +157,186 @@ public function update($options = ['langcodes' => self::REQ])
 
         drush_backend_batch_process();
     }
+
+    /**
+     * Imports to a gettext translation file.
+     *
+     * @command locale:import
+     * @validate-module-enabled locale
+     * @param $langcode The language code of the imported translations.
+     * @param $file Path and file name of the gettext file.
+     * @option type The type of translations to be imported, defaults to 'not-customized'. Options:
+     *   - customized: Treat imported strings as custom translations.
+     *   - not-customized: Treat imported strings as not-custom translations.
+     * @option override Whether and how imported strings will override existing translations. Defaults to the Import behavior configurred in the admin interface. Options:
+     *  - none: Don't overwrite existing translations. Only append new translations.
+     *  - customized: Only override existing customized translations.
+     *  - not-customized: Only override non-customized translations, customized translations are kept.
+     *  - all: Override any existing translation.
+     * @usage drush locale-import nl drupal-8.4.2.nl.po
+     *   Import the Dutch drupal core translation.
+     * @usage drush locale-import nl custom-translations.po --type=custom --override=all
+     *   Import customized Dutch translations and override any existing translation.
+     * @aliases locale-export
+     * @throws \Exception
+     */
+    public function import($langcode, $file, $options = ['type' => self::OPT, 'override' => self::OPT])
+    {
+        if (!file_exists($file)) {
+            throw new \Exception(dt('File @file not found.', ['@file' => $file]));
+        }
+
+        $language = $this->getTranslatableLanguage($langcode, true);
+
+        $this->getModuleHandler()->loadInclude('locale', 'translation.inc');
+        $this->getModuleHandler()->loadInclude('locale', 'bulk.inc');
+
+        $translationOptions = _locale_translation_default_update_options();
+        $translationOptions['langcode'] = $language->getId();
+        $translationOptions['customized'] = $this->convertCustomizedType($options['type']);
+        $override = $this->convertOverrideOption($options['override']);
```

Comment:
I'm missing the override settings from /admin/config/regional/translate/settings

Reply:
All override options are covered by `override`. If no override is provided the ones from `_locale_translation_default_update_options()` are used.